app/models/content_store/content_item.rb

```diff
@@ -0,0 +1,54 @@
+class ContentItem
```

Comment:
I think it'd be make sense for this class to be within a content_store module, would save having to tweak the autoloader and feels less surprising.